modules/reporting/csv.py

```diff
@@ -37,6 +37,6 @@ def module_run(self):
                     for cell in row:
                         if cell and cell[0] in badcharacters:
                             cell = ' '+cell
-                        sanitized_row.append(cell.encode("utf-8"))
+                        sanitized_row.append(cell.encode("utf-8").decode())
```

Comment:
Correct me if I'm wrong, but isn't this the same thing as `sanitized_row.append(cell)`? You're encoding then re-decoding back to the original encoding. Please test it an modify as needed.

Reply:
Ya not really sure what I was thinking there but fixed it up.